### ISSUE TITLE
Crash after loading webview

### ISSUE THREAD

**Issue by frozeman** [atom/electron#2031]
My electron just crashed after loading a web view by changing its source. 

```
...
[42780:0622/154523:INFO:renderer_main.cc(200)] Renderer process started
[36037:0622/154523:FATAL:render_frame_host_manager.cc(1422)] Check failed: render_frame_host_->GetSiteInstance() != instance (0x7fae0b8558a0 vs. 0x7fae0b8558a0)
[0622/154523:ERROR:http_transport_mac.mm(186)] Could not connect to the server. (NSURLErrorDomain -1004)
[42780:0622/154528:WARNING:channel.cc(315)] RawChannel write error
[42015:0622/154528:WARNING:channel.cc(315)] RawChannel write error
```

I'm using preload scripts, maybe some race condition here killed it.

**Comment by deepak1556** [comment]
a sample test case that causes the crash would help.

**Comment by frozeman** [comment]
Will try if i can reproduce it.

**Comment by frankhale** [comment]
Which version of Electron are you using?

**Comment by frozeman** [comment]
0.28.2 was it i guess

**Comment by zcbenz** [comment]
It should have been fixed v0.29.2.
